django/db/models/base.py

```diff
@@ -1193,7 +1193,8 @@ def check(cls, **kwargs):
             clash_errors = (
                 cls._check_id_field() +
                 cls._check_field_name_clashes() +
-                cls._check_model_name_db_lookup_clashes()
+                cls._check_model_name_db_lookup_clashes() +
```

Comment:
I think this can be:
```
clash_errors = (
    *cls._check_id_field(),
    *cls._check_field_name_clashes(),
    *cls._check_model_name_db_lookup_clashes(),
    *cls._check_property_name_foreign_key_id_accessor_clashes(),
)
```
correct, @ngnpope?

Reply:
Yes, correct.